### ISSUE TITLE
`padding='causal'` is in document but cannot be used with `tf.keras.layers.Conv1D`

### ISSUE THREAD

**Issue by dhgrs** [tensorflow/tensorflow#21763]
### System information
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**:
Yes
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:
Linux Ubuntu 16.04
- **Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device**:
- **TensorFlow installed from (source or binary)**:
I use docker tensorflow/tensorflow:1.10.0-devel-gpu-py3
- **TensorFlow version (use command below)**:
1.10.0
- **Python version**:
Python 3.5.2
- **Bazel version (if compiling from source)**:
- **GCC/Compiler version (if compiling from source)**:
- **CUDA/cuDNN version**:
- **GPU model and memory**:
- **Exact command to reproduce**:
see my gist below

### Describe the problem
In the [document](https://www.tensorflow.org/versions/r1.10/api_docs/python/tf/keras/layers/Conv1D), we can use `causal` option for  `padding`. But `causal` option causes error in r1.10. We should remove `causal` from the document or support `causal`. There are related issues, #14933, #15000 and #15037. But all of them are closed. I think they should be reopen.

### Source code / logs
https://gist.github.com/dhgrs/ca552f5804ddfb4db9669f9189eba76f

**Comment by tensorflowbutler** [comment]
Nagging Assignee @drpngx: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @drpngx: It has been 29 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by omalleyt12** [comment]
This is fixed now in the latest nightly version, please ```pip install tf-nightly``` (or ```pip install tf-nightly-gpu``` for GPU support)
